# DLMCBP
A Brief Survey of Deep Learning-based Models for CircRNA-Protein Binding Sites Prediction

# CircRNA-RBP binding database
CircIntercome

Provides the genomic location of RBP binding sites in circRNAs sequence

https://circinteractome.nia.nih.gov/index.html

CSCD2.0 

Records the genomic location, function, and regulation of cancer-associated circRNAs, predicted MREs, RBPs and ORFs

http://gb.whu.edu.cn/CSCD/

TSCD 

Record whether circRNA binds to RBP

http://gb.whu.edu.cn/TSCD/

CircAtlas 3.0 

Provides details about conservation across species, interactions between circRNA and miRNA, predicted IRESs, ORFs and RBPs

https://ngdc.cncb.ac.cn/circatlas/

ENCORI 

Records a large amount of biological molecule interaction data about miRNA-ncRNA, miRNA-mRNA, RBP-RNA and RNA-RNA 

https://rnasysu.com/encori/

# CircRNA-related annotation database
circBase

Provides circRNA genomic information of five species, including genomic position, sequences, and gene symbols 

http://www.circbase.org/

circRNADb

Provides human circRNAs with protein-coding annotations

http://reprod.njmu.edu.cn/circrnadb

CircBank

Provides a novel nomenclature system based on the host gene name, chromosome location, and collects circRNA conservation, miRNA binding, protein coding potential, methylation, etc.

http://www.circbank.cn/

CIRCpedia V2

Provides annotations for circular RNAs from over 180 RNA-seq datasets across six different species

https://www.picb.ac.cn/rnomics/circpedia/

CircNet 2.0

Records the tissue-specific circRNA expression profiles and circRNA-miRNA-gene regulatory network

https://awi.cuhk.edu.cn/~CircNet

CircRiC

Provides the modules of integrative analysis, drug response, biogenesis, and expression landscape

https://hanlab.uth.edu/cRic/

CircFunBase

Records circRNA function, interactions between circRNA and miRNA, and GO annotations 

https://bis.zju.edu.cn/CircFunBase/

TRCirc

Provides details about over 765 000 transcription factor-circRNA relationships

https://bio.liclab.net/TRCirc

deepBase V3.0

Provides the annotations and comprehensive expression analysis of circRNAs, lncRNAs, tRFs, and sRNAs for 14 species

https://rna.sysu.edu.cn/deepBase

# Protein-related annotation database
Uniprot

Provides the high-quality, comprehensive, and freely available resource of protein sequence and functional information

https://www.uniprot.org/

String

Provides protein-protein interaction networks and functional enrichment analysis

https://cn.string-db.org/

NCBI Protein

Records protein annotations collected from other database, like GenBank, RefSeq, TPA, etc.

https://www.ncbi.nlm.nih.gov/protein/

HPRD 9.0

Provides the information of domain architecture, post-translational modifications, interaction networks and disease association for each protein in the human proteome

http://www.hprd.org/

RBPWorld/EuRBPDB V2

Provides precise and comprehensive list of RBPs of each eukaryote with annotations collected from various public databases.

http://research.gzsys.org.cn/eurbpdb2/index.html

# Deep Learning-based Models for CircRNA-Protein Binding Sites Prediction
CRIP

K. Zhang, X. Pan, Y. Yang, H.-B. Shen, CRIP: predicting circRNA–RBP-binding sites using a codon-based encoding and hybrid deep neural networks, Rna, 25 (2019) 1604-1615. DOI: 10.1261/rna.070565.119 

https://github.com/kavin525zhang/CRIP

CircSLNN

Y. Ju, L. Yuan, Y. Yang, H. Zhao, CircSLNN: identifying RBP-binding sites on circRNAs via sequence labeling neural networks, Frontiers in genetics, (2019) 1184. DOI: 10.3389/fgene.2019.01184

https://github.com/JuYuqi/circSLNN

CSCRSites

Z. Wang, X. Lei, F.-X. Wu, Identifying cancer-specific circRNA–RBP binding sites based on deep learning, Molecules, 24 (2019) 4035. DOI: 10.3390/molecules24224035

MFNN

Z. Wang, X. Lei, Matrix factorization with neural network for predicting circRNA-RBP interactions, BMC bioinformatics, 21 (2020) 1-15. DOI: 10.1186/s12859-020-3514-x

PASSION

C. Jia, Y. Bi, J. Chen, A. Leier, F. Li, J. Song, PASSION: an ensemble neural network approach for identifying the binding sites of RBPs on circRNAs, Bioinformatics, 36 (2020) 4276-4282. DOI: 10.1093/bioinformatics/btaa522

http://flagship.erc.monash.edu/PASSION/

DeCban
L. Yuan, Y. Yang, DeCban: prediction of circRNA-RBP interaction sites by using double embeddings and cross-branch attention networks, Frontiers in Genetics, 11 (2021) 632861. DOI: 10.3389/fgene.2020.632861

https://github.com/AaronYll/DECban

iCircRBP-DHN

Y. Yang, Z. Hou, Z. Ma, X. Li, K.-C. Wong, iCircRBP-DHN: identification of circRNA-RBP interaction sites using deep hierarchical network, Briefings in Bioinformatics, 22 (2021) bbaa274. DOI: 10.1093/bib/bbaa274 

https://github.com/houzl3416/iCircRBP-DHN


circRB

Z. Wang, X. Lei, Identifying the sequence specificities of circRNA-binding proteins based on a capsule network architecture, BMC bioinformatics, 22 (2021) 1-16. DOI: 10.1186/s12859-020-03942-3 

https ://github.com/wzf171/circRB

CRPBsites

Z. Wang, X. Lei, Prediction of RBP binding sites on circRNAs using an LSTM-based deep sequence learning architecture, Briefings in Bioinformatics, 22 (2021) bbab342. DOI: 10.1093/bib/bbab342 

https://github.com/wzf171/CRPBsites

iDeepC

H. Wu, X. Pan, Y. Yang, H.-B. Shen, Recognizing binding sites of poorly characterized RNA-binding proteins on circular RNAs using attention Siamese network, Briefings in bioinformatics, 22 (2021) bbab279. DOI: 10.1093/bib/bbab279

https://github.com/hehew321/iDeepC

DMSK

H. Li, Z. Deng, H. Yang, X. Pan, Z. Wei, H.-B. Shen, K.-S. Choi, L. Wang, S. Wang, J. Wu, circRNA-binding protein site prediction based on multi-view deep learning, subspace learning and multi-view classifier, Briefings in Bioinformatics, 23 (2022) bbab394.  DOI: 10.1093/bib/bbab394

https://github.com/Rebecca3150/DMSK

HCRNet

Y. Yang, Z. Hou, Y. Wang, H. Ma, P. Sun, Z. Ma, K.-C. Wong, X. Li, HCRNet: high-throughput circRNA-binding event identification from CLIP-seq data using deep temporal convolutional network, Briefings in Bioinformatics, 23 (2022). DOI: 10.1093/bib/bbac027

https://github.com/yangyn533/HCRNet

CRBPDL

M. Niu, Q. Zou, C. Lin, CRBPDL: Identification of circRNA-RBP interaction sites using an ensemble neural network approach, PLoS computational biology, 18 (2022) e1009798. DOI: 10.1371/journal.pcbi.1009798 

https://github.com/nmt315320/CRBPDL

circRIP

X. Dong, K. Chen, W. Chen, J. Wang, L. Chang, J. Deng, L. Wei, L. Han, C. Huang, C. He, circRIP: an accurate tool for identifying circRNA–RBP interactions, Briefings in Bioinformatics, 23 (2022) bbac186. DOI: 10.1093/bib/bbac186

https://github.com/bioinfolabwhu/circRIP

CircSite

H. Wu, Y. Fang, Y. Yang, X. Pan, H.-B. Shen, Decoding protein binding landscape on circular RNAs with base-resolution Transformer models, bioRxiv, (2022) 2022.2011. 2020.517239. DOI: 10.1101/2022.11.20.517239

http://www.csbio.sjtu.edu.cn/bioinf/CircSite

circ-pSBLA

Y. Guo, X. Lei, A pseudo-Siamese framework for circRNA-RBP binding sites prediction integrating BiLSTM and soft attention mechanism, Methods, 207 (2022) 57-64. DOI: 10.1016/j.ymeth.2022.09.003

https://github.com/gyj9811/circ-pSBLA.

CRWS/ CirRBP

Z. Wang, X. Lei, A web server for identifying circRNA-RBP variable-length binding sites based on stacked generalization ensemble deep learning network, Methods, 205 (2022) 179-190. DOI: 10.1016/j.ymeth.2022.06.014

http://www.bioinformatics.team

JLCRB

X. Du, Z. Xue, JLCRB: A unified multi-view-based joint representation learning for CircRNA binding sites prediction, Journal of Biomedical Informatics, 136 (2022) 104231. DOI: 10.1016/j.jbi.2022.104231

https://github.com/Xuezg/JLCRB

CircSSNN

C. Cao, S. Yang, M. Li, C. Li, CircSSNN: circRNA-binding site prediction via sequence self-attention neural networks with pre-normalization, BMC bioinformatics, 24 (2023) 220. DOI: 10.1186/s12859-023-05352-7

https://github.com/ cc646201081/CircSSNN

circ2CBA

Y. Guo, X. Lei, L. Liu, Y. Pan, circ2CBA: prediction of circRNA-RBP binding sites combining deep learning and attention mechanism, Frontiers of Computer Science, 17 (2023) 175904. DOI:10.1007/s11704-022-2151-0

CRMSS

L. Zhang, C. Lu, M. Zeng, Y. Li, J. Wang, CRMSS: predicting circRNA-RBP binding sites based on multi-scale characterizing sequence and structure features, Briefings in Bioinformatics, 24 (2023) bbac530. DOI: 10.1093/bib/bbac530

https://github.com/BioinformaticsCSU/CRMSS


ASCRB

L. Li, Z. Xue, X. Du, ASCRB: Multi-view based attentional feature selection for CircRNA-binding site prediction, Computers in Biology and Medicine, (2023) 107077. DOI: 10.1016/j.compbiomed.2023.107077

https://github.com/dxqllp/ASCRB
